src/main/java/it/smartcommunitylab/aac/webauthn/service/WebAuthnRpService.java

```diff
@@ -133,6 +142,24 @@ public WebAuthnRpService(RelyingParty rp,
         return Optional.empty();
     }
 
+    private String convertTransportsToString(Set<AuthenticatorTransport> transports) {
+        final List<String> result = new LinkedList<>();
+        for (final AuthenticatorTransport t : transports) {
+            if (t == AuthenticatorTransport.USB) {
+                result.add("USB");
```

Comment:
avoid hardcoding this kind of conversion, use a method from the class. AuthenticationTransport.toJsonString should work.
https://github.com/Yubico/java-webauthn-server/blob/fe7a55710d1219ea03c514de872d95de0a15eb45/webauthn-server-core/src/main/java/com/yubico/webauthn/data/AuthenticatorTransport.java#L155

Reply:
Done in 674587d6d918161c1fd98065e2cef0484a6f55da (using `getId` instead of `toJsonString` since they both end up returning the id)